packages/rs-dpp/src/data_contract/document_type/random_index.rs

```diff
@@ -0,0 +1,57 @@
+use crate::data_contract::document_type::{Index, IndexProperty};
+use crate::ProtocolError;
+use rand::prelude::StdRng;
+use rand::seq::SliceRandom;
+use rand::Rng;
+
+impl Index {
```

Comment:
Should we lock it behind a feature, so it won't slip into the end build? It's not a big issue for the server side, but for the SDK that would be quite a problem

Reply:
made a task